MCprep_addon/materials/prep.py

```diff
@@ -489,19 +494,7 @@ def execute(self, context):
 			generate.set_saturation_material(mat)
 
 		if self.prepMaterials:
-			bpy.ops.mcprep.prep_materials(
-				animateTextures=self.animateTextures,
-				autoFindMissingTextures=self.autoFindMissingTextures,
-				combineMaterials=self.combineMaterials,
-				improveUiSettings=self.improveUiSettings,
-				usePrincipledShader=self.usePrincipledShader,
-				useReflections=self.useReflections,
-				useExtraMaps=self.useExtraMaps,
-				normalIntensity=self.normalIntensity,
-				makeSolid=self.makeSolid,
-				syncMaterials=self.syncMaterials,
-				packFormat=self.packFormat,
-				skipUsage=True)
+			self.prep_mat(context)
```

Comment:
This line is definitely a big pro of this change. With the old way, it is tech debt of constantly having to keep the same args in sync in different places.